oak_functions/loader/src/server.rs

```diff
@@ -32,6 +32,48 @@ pub(crate) struct WasmHandler {
     module: Arc<wasmi::Module>,
 }
 
+/// Encapsulates the state of a Wasm invocation for a single user request.
+struct WasmState {
+    instance: wasmi::ModuleRef,
+    #[allow(dead_code)]
+    memory: wasmi::MemoryRef,
```

Comment:
Why is memory needed as a separate filed in the state? It can already be extracted from `instance`.

Reply:
It is a similar patter to the Oak Runtime. When we implement the ABI on this struct, it will be simpler if the memory is already available as a separate field.